src/_settings.scss

```diff
@@ -474,3 +475,13 @@ $size-topbar-user-toggle-icon: 24px !default;
 $w-topbar-user-nav: 90% !default;
 $maw-topbar-user-nav: 350px !default;
 $bgc-topbar-user-cta: $c-accent !default;
+
+// Thumbnail
+$bgc-thumbnail: $c-white !default;
+$bd-thumbnail: 1px solid $c-gray-lightest !default;
+$p-thumbnail: ceil($p-base / 6) !default;
```

Comment:
What do you think about using `$p-v-xsmall $p-h-xsmall` instead of `$p-base / 6`?
so we'll concentrate all paddings in the main variables.